content/kubermatic/master/advanced/custom_ui/_index.en.md

```diff
@@ -5,43 +5,73 @@ weight = 2
 pre = "<b></b>"
 +++
 
-This manual explains how to apply custom theme to the application. As we are using SCSS that needs to be processed into
-the CSS during the build, there are different approaches to it that will be described in the following sections.
+This manual explains multiple approaches to add custom themes to the application. It also describes how to add and enable multiple themes for the ui.
 
-## Customizing the Application Sources
+Here are some quick links to the different chapters:
 
-This is the most obvious approach to customizing the application. However, it requires access to the repository as some
-of the sources need modifications. Its biggest advantage is possibility to reuse rules that are already defined.
+* [Modifying Available Themes]({{< ref "#modifying-available-themes" >}})
+* [Disabling Theming Functionality]({{< ref "#disabling-theming-functionality" >}})
+* [Possible Customizing Approaches]({{< ref "#possible-customizing-approaches" >}})
+  * [Customizing the Application Sources]({{< ref "#customizing-the-application-sources" >}})
+  * [Customizing the Application Sources Inside Custom Container]({{< ref "#customizing-the-application-sources-inside-custom-container" >}})
+  * [Customizing the Application Without Changing the Sources]({{< ref "#customizing-the-application-without-changing-the-sources" >}})
 
-All the application-wide rules are stored inside `src/assets/css` directory, where `_main.scss` is the main file that
-imports all other files that are required. The recommended approach to override application default styling is to
-register custom CSS files inside `_main.scss` and override existing CSS files only if it is needed. Overriding existing
-files is required when for example color variables need to be changed application-wide.
+## Modifying Available Themes
+Currently the application has two themes by default: light and dark. Each user can specify the theme that he want to use in the `User Settings` view which is accessible from the user menu.
```

Comment:
Do we need the screenshot of the UI here to show a theme switcher?

 > ... Each user can specify the theme that he want to use ...

s/want/wants

Maybe also change to "he/she" or simply change whole sentence to "Every user can select the theme in the `User Settings`..."